packages/kit/src/exports/vite/dev/index.js

```diff
@@ -51,18 +54,24 @@ export async function dev(vite, vite_config, svelte_config) {
 	async function update_manifest() {
 		try {
 			({ manifest_data } = await sync.create(svelte_config));
+
+			if (manifest_error) {
+				manifest_error = null;
+				vite.ws.send({ type: 'full-reload' });
+			}
 		} catch (error) {
-			console.error(colors.bold().red('Failed to update manifest'));
+			manifest_error = /** @type {Error} */ (error);
+
+			console.error(colors.bold().red('Invalid routes'));
```

Comment:
This could be more than invalid routes (matchers for example) - I chose "manifest" because it was a more generic while not being completely opaque to the user

Reply:
I think it's still okay — matchers _are_ part of your routes